README.md

```diff
@@ -19,21 +19,27 @@ This quickstart requires a working Docker Desktop installation and git:
     git clone https://github.com/cfpb/consumerfinance.gov.git
     cd consumerfinance.gov
     ```
+- One of the following runtimes:
+  - [Set up and run the Docker containers via docker-compose](https://cfpb.github.io/consumerfinance.gov/installation/#set-up-and-run-the-docker-containers):
```

Comment:
I think we need to have some D&D discussions around all of this. While k8s is great, it does introduce more complexities that aren't necessarily something needed to be taken on for a local development purpose. Until we better understand the intentions of who is going to have access to k8s, I think we will still need to support it to a degree. 

Having said all of that, I have done my best to give as much of a 1:1 to how docker-compose is currently implemented, and its all there as far as I can tell. However, this did bring in the issue of k8s not liking anything to do with relative paths. So for our stuff to be a nice single "up" command (`helm-install.sh` or otherwise), there has to be some non-vanilla scripting that finds the project directory's absolute path and injects it into the overrides with `envsubst` in `helm-install.sh`.  Also, k8s has no concept of building an image either, hence `build-images.sh` building through `docker` directly. Just some considerations to take into account when making the decision to take out docker-compose.